pom.xml

```diff
@@ -103,9 +104,19 @@
     <repositories>
         <repository>
             <id>snapshot-repository</id>
-            <name>Maven2 Snapshot Repository</name>
+            <name>Hazelcast Snapshot Repository</name>
             <url>https://oss.sonatype.org/content/repositories/snapshots</url>
         </repository>
+        <repository>
+            <id>main-repo</id>
+            <name>Main Repository</name>
+            <url>https://repo1.maven.org/maven2</url>
+        </repository>
+        <repository>
+            <id>backup-repo</id>
+            <name>Backup Repository</name>
+            <url>https://repo2.maven.org/maven2/</url>
+        </repository>
```

Comment:
Why did you add these?

Reply:
This came from https://github.com/hazelcast/hazelcast-wm/commit/9fb91a4ae7a939c4db49d2d87a041dd71c33329f. So @Fly-Style, can you weigh in?

Reply:
I thought they were required, but if not, let me delete them